src/scraper/utils.py

```diff
@@ -0,0 +1,42 @@
+"""
+    Misc. helper functions.
+"""
+from bs4 import BeautifulSoup
+
+
+def clean(element: str) -> str:
```

Comment:
```suggestion
def strip_html_tags(element: str) -> str:
```
better to go with a name that matches exactly what this does, clean seems too broad in my opinion. 
What do you think?

Reply:
You're totally right, there's no good reason to leave it as just "clean".

Reply:
advice: after you write tests for this function/module, you'll (probably) have a better guess of what the best name would be.